app/controllers/admins_controller.rb

```diff
@@ -64,9 +60,8 @@ def verify
     if voter.save
       send_email = params[:send_email] == "true"
       if send_email
-        # Will need to be replaced with deliver_now
         begin
-          UserMailer.voter_verified(voter, event_year).deliver
+          UserMailer.voter_verified(voter, event_year).deliver_now
```

Comment:
Rails is just a really big framework for writing web applications using ruby (it definitely isn't the only ruby web framework and people certainly write non-web code in ruby).

The version of rails for this project was updated in  [https://github.com/ywwg/ffagc/pull/60/files#diff-8b7db4d5cc4b8f6dc8feb7030baa2478R5](#60) and is now `4.2.7.1`. In general you can check the version of a gem used by a project by looking at `Gemfile.lock`. For rails in particular running `bundle exec rails -v` will print the version.